bibtex-actions-file.el

```diff
@@ -68,6 +74,19 @@ If you use 'org-roam' and 'org-roam-bibtex, you should use
      (mapcar
       (lambda (p) (file-truename p)) file-paths))))
 
+(defun bibtex-actions-file-parser-default (dirs file-field)
+  "Return a list of files from base directories DIRS and FILE-FIELD."
+  (seq-map (lambda (dir) (expand-file-name file-field dir)) dirs))
+
+(defun bibtex-actions-file-parser-zotero (dirs file-field)
+  "Return a list of files from DIRS and a Zotero formatted FILE-FIELD."
+  (cl-loop
+   for dir in dirs
+   for field in (split-string file-field ";")
+   for filename = (expand-file-name field dir)
+   if (file-regular-p filename)
```

Comment:
I had recently removed use of `cl-loop` in favor of the `seq` and `dolist` functions for consistency, but I can't figure out an elegant way to do that here.

Should this be "when" though, since there's no "then" clause?

```suggestion
   when (file-regular-p filename)
```